x-pack/plugin/esql/qa/testFixtures/src/main/resources/semantic_text.csv

```diff
@@ -0,0 +1,4 @@
+semantic_text_field:semantic_text
```

Comment:
I wonder if `SEMANTIC_TEXT` makes sense for multi-valued field? It might be nice to have some multi-valued data to experiment with.

Reply:
Yes it absolutely does! I was not sure whether I should add a test for it here, but I did anyway for `MV_EXPAND`